mini_programs/simple_calculator.py

```diff
@@ -24,26 +25,24 @@ def final_result(n, m):
         result = subtract(n, m)
```

Comment:
```suggestion
        result = n - m
```
I think it’s better doing this rather than writing separate functions.
And same for add, multiply and divide operations as well.

Reply:
I agree. I'll change it soon.